README.rst

```diff
@@ -183,30 +183,101 @@ All
 Transactions
 ''''''''''''
 
+Along with the required account ID, you may also
+pass the following optional arguments:
+
+* ``cursor`` (string): An API cursor to fetch a specific set of results.
+* ``start`` (ISO-8601 datetime string): Fetch transactions after this time.
+* ``end`` (ISO-8601 datetime string): Fetch transactions before this time.
+
 .. code:: python
 
     from pybutton import Client
 
     client = Client('sk-XXX')
 
-    response = client.accounts.transactions('acc-123',
+    response = client.accounts.transactions(
+        'acc-123',
         start='2016-07-15T00:00:00.000Z',
         end='2016-09-30T00:00:00.000Z'
     )
 
     print(response)
     # <class pybutton.Response [100 elements]>
 
-    cursor = response.next()
+Response
+--------
 
-    response = client.accounts.transactions('acc-123',
-        cursor=cursor,
-        start='2016-07-15T00:00:00.000Z',
-        end='2016-09-30T00:00:00.000Z'
-    )
+The format of the ``pybutton.Response`` class varies depending on whether it contains
+one or multiple elements.
+
+Methods
+~~~~~~~
+
+Data
```

Comment:
because these are methods, let's leave them lowercased.  A little weird, but the sections above are conceptually "Create", not meaning to refer to the method name.